<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="3" column="3">
       <widget class="QLineEdit" name="ExceptionList">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="CheckpointSwitch">
        <property name="text">
         <string>Контрольные значения</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="CheckpointList">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="ExceptionSwitch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Исключения</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="CommandList"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="StartValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="FinalValue"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="FinalLabel">
        <property name="text">
         <string>Конечное значение</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="CommandLabel">
        <property name="text">
         <string>Команды</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="StartLabel">
        <property name="text">
         <string>Начальное значение</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="CalculateButton">
      <property name="text">
       <string>Посчитать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ExceptionSwitch</sender>
   <signal>toggled(bool)</signal>
   <receiver>ExceptionList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckpointSwitch</sender>
   <signal>toggled(bool)</signal>
   <receiver>CheckpointList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
